testsuite/MathMLToDisplay/issue860-2.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>Test ignoreMMLattributes configuration option</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../header.js"></script>
+    <script type="text/javascript" src="../scriptTests.js"></script>
+    <script type="text/javascript">
+      function preMathJax() {
+        gConfigObject.ignoreMMLattributes = {'data-skip': true};
+      }
+      function getTestCases() {
+        var math = document.getElementById('MathJax-Element-1-Frame');
+        return [
+          newScriptReftestResult("Check if data-skip attribute is not preserved",
+          !(math.innerHTML.indexOf('data-skip') > -1))
```

Comment:
A little unusual to use `!(x > y)` when `(x <= y)` (or in this case, `(x === y)`) would work.  But I'll merge it anyway.